src/components/coin-divider/CoinDivider.js

```diff
@@ -66,6 +70,14 @@ export default function CoinDivider({
     toggleOpenSmallBalances,
   } = useOpenSmallBalances();
 
+  const toggleSmallBalances = useCallback(() => {
+    if (!isSmallBalancesOpen && !fetchedCharts) {
+      dispatch(emitChartsRequest());
```

Comment:
can we maybe be more explicit and provide assets here? This will make this logic a bit more readable